tests/codebuild/run_canarytest.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-source tests/codebuild/common.sh
+source ./common.sh
```

Comment:
You mentioned something about `set -e` flag in chat.. would that have helped us catch this // should we set it?

Reply:
It's not shown in this pull request preview, but we `set -e` on line 46 after we create the cleanup trap. I will move this source down below that to catch this in the future.

Reply:
Hmm I see how that would make "imports" hairy.